app/src/main/java/my/noveldokusha/scraper/sources/Aixdzs.kt

```diff
@@ -0,0 +1,116 @@
+package my.noveldokusha.scraper.sources
+
+import android.net.Uri
+import my.noveldokusha.data.BookMetadata
+import my.noveldokusha.data.ChapterMetadata
+import my.noveldokusha.scraper.*
+import org.jsoup.nodes.Document
+
+class Aixdzs : SourceInterface.Catalog {
+    override val name = "Aixdzs"
+    override val baseUrl = "https://www.aixdzs.com/"
+    override val catalogUrl = "https://www.aixdzs.com/sort/1/index.html"
+    override val language = "Chinese"
+
+    override suspend fun getChapterTitle(doc: Document): String? = null
+
+    override suspend fun getChapterText(doc: Document): String {
+        return doc
+            .selectFirst(".content")!!
+            .let { textExtractor.get(it) }
+    }
+
+    override suspend fun getBookCoverImageUrl(doc: Document): String? {
+        return doc
+            .selectFirst(".d_af.fdl")
+            ?.selectFirst("img[src]")
+            ?.attr("src")
+
+    }
+
+
+    override suspend fun getBookDescripton(doc: Document): String? {
+        return doc
+            .selectFirst(".d_co")
+            ?.let { textExtractor.get(it) }
+    }
+
+    override suspend fun getChapterList(doc: Document): List<ChapterMetadata> {
+        return doc.select("div#i-chapter a[href]").map {
+            ChapterMetadata(title = it.text(), url = baseUrl + it.attr("href"))
+        }
+    }
+
+    override suspend fun getCatalogList(index: Int): Response<PagedList<BookMetadata>> {
+        val page = index + 1
+//        val url = baseUrl.toUrlBuilderSafe().apply {
+//            if (page == 1) addPath("all.html")
+//            else addPath("all-$page.html")
+//        }
+
```

Comment:
Remove commented code.

Reply:
that code is reminder to add next page with that setting.